src/app/modules/ix-forms/components/ix-icon-group/ix-icon-group.component.html

```diff
@@ -0,0 +1,28 @@
+<ix-label
+  *ngIf="label || tooltip"
+  [label]="label"
+  [tooltip]="tooltip"
+  [required]="required"
+></ix-label>
+
+<div class="icon-group" [attr.aria-label]="label">
+  @for (option of options; track option) {
```

Comment:
for me adding tabbing is more readable.
```
  @for (option of options; track option) {
    <button
      mat-icon-button
      type="button"
      [disabled]="isDisabled"
      [ixTest]="[controlDirective.name, option[0]]"
      [attr.aria-label]="option[0]"
      [class.selected]="value === option[0]"
      (click)="onValueChanged(option[0])"
      (blur)="onTouch()"
    >
      <ix-icon [name]="option[1]"></ix-icon>
    </button>
  }
  @empty {
    <span>{{ 'No options are passed' | translate }}</span>
  }
```
But in docs they have not added tabbing as well.. https://angular.io/api/core/for

Reply:
This code follows existing linter rules.